aws/resource_aws_codepipeline_test.go

```diff
@@ -435,7 +414,7 @@ func TestAccAWSCodePipeline_multiregion_ConvertSingleRegion(t *testing.T) {
 				),
 			},
 			{
-				Config: testAccAWSCodePipelineConfig_backToBasic(name, githubToken),
+				Config: testAccAWSCodePipelineConfig_backToBasic(name),
```

Comment:
Not sure if related to changes here, but this test step is failing for me locally and in TeamCity:

```
=== CONT  TestAccAWSCodePipeline_multiregion_ConvertSingleRegion
    resource_aws_codepipeline_test.go:380: Step 3/4 error: Error running apply:
        Error: region cannot be set for a single-region CodePipeline


--- FAIL: TestAccAWSCodePipeline_multiregion_ConvertSingleRegion (62.80s)
```

Reply:
This is a pre-existing error with Terraform 0.14, though it passes with Terraform 0.12.29

Reply:
Heh, I created #16706 in December